### PR TITLE
DGGRID tool for Dicrete Global Grid Systems

### PR DESCRIPTION
**Pull request by allixender**
The tool DGGRIDv7 by Kevin Sahr for use multi-platform in Julia

[https://github.com/sahrk/DGGRID](https://github.com/sahrk/DGGRID)

with some patches to make the build also work on mingw/Windows

Other popular DGGIRD bundles are in R and Python

- https://github.com/r-barnes/dggridR
- https://github.com/allixender/dggrid4py/

### CODE REVIEW

File: D/DGGRID7/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf),
+    MacOS(:x86_64),
+    FreeBSD(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
+]
```
Comment (giordano) [inline D/DGGRID7/build_tarballs.jl:37]:
```suggestion
platforms = supported_platforms()
```

**Comment by giordano** [comment]
https://github.com/allixender/DGGRID/blob/0dabdfd3f8fd6ac27626064a94c483e610d578bb/src/MakeIncludes#L18-L20 quite frustratingly forces `gcc` and `g++` as compilers, without the possibility of overriding them with environment variables without patching the file, and they don't even used standard environment variables like `CC` and `CXX`

**Comment by allixender** [comment]
Thanks a lot.

**Comment by giordano** [comment]
Thanks for your contribution :slightly_smiling_face:

**Pull request merged**